lib/alchemy/page_layout.rb

```diff
@@ -38,7 +38,7 @@ def add(page_layout)
       def get(name)
         return {} if name.blank?
 
-        all.detect { |a| a["name"].casecmp(name).zero? }
+        all.detect { |a| a["name"].casecmp(name).zero? }&.with_indifferent_access
```

Comment:
Could we instead add this to the `all` method?

```rb
def all
  @definitions ||= read_definitions_file.map(&:with_indifferent_hash)
end
```

Reply:
Also, I would prefer this to be a separate commit with the explanation on the commit message.